hack/make/release-rpm

```diff
@@ -14,6 +14,7 @@ set -e
 #
 # ... and so on and so forth for the builds created by hack/make/build-rpm
 
+source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
```

Comment:
Shouldn't be needed `.detect-daemon-osarch` takes the client version info if the daemon is not started